src/PodActions.jsx

```diff
@@ -162,31 +168,31 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                           })}
                               component="button">
                 {_("Pause")}
-            </DropdownItem>,
+            </DropdownItemDeprecated>,
         );
     }
 
     if (dropdownItems.length > 1) {
-        dropdownItems.push(<DropdownSeparator key="separator-1" />);
+        dropdownItems.push(<DropdownSeparatorDeprecated key="separator-1" />);
     }
     dropdownItems.push(
-        <DropdownItem key="action-delete"
+        <DropdownItemDeprecated key="action-delete"
                           className="pod-action-delete pf-m-danger"
                           onClick={() => {
                               Dialogs.show(<PodDeleteModal pod={pod} />);
                           }}
                           component="button">
             {_("Delete")}
-        </DropdownItem>,
+        </DropdownItemDeprecated>,
     );
 
     if (!dropdownItems.length)
         return null;
 
     return (
-        <Dropdown onSelect={() => setOpen(!isOpen)}
-                         position={DropdownPosition.right}
-                         toggle={<KebabToggle onToggle={(value, _) => setOpen(value)} id={"pod-" + pod.Name + (pod.isSystem ? "-system" : "-user") + "-action-toggle"} />}
+        <DropdownDeprecated onSelect={() => setOpen(!isOpen)}
+                         position={DropdownPositionDeprecated.right}
+                         toggle={<KebabToggleDeprecated onToggle={(_event, value, _) => setOpen(value)} id={"pod-" + pod.Name + (pod.isSystem ? "-system" : "-user") + "-action-toggle"} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1266-20230413-122810-fbb171ea-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)